<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TabGridWidget</class>
 <widget class="QWidget" name="TabGridWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>710</width>
    <height>455</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="btnAddSilence">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>316</y>
     <width>261</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Add silent slice</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblSliceCOunt">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>430</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="btnRemove">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>421</y>
     <width>81</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>&amp;Remove</string>
   </property>
   <property name="shortcut">
    <string>Meta+D</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnAddWav">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>421</y>
     <width>81</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>&amp;Add</string>
   </property>
   <property name="shortcut">
    <string>Meta+A</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnPlay">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>421</y>
     <width>81</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>&amp;Play</string>
   </property>
   <property name="shortcut">
    <string>Meta+P</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>10</y>
     <width>91</width>
     <height>101</height>
    </rect>
   </property>
   <property name="title">
    <string>Samplerate</string>
   </property>
   <widget class="QRadioButton" name="radio44">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>71</width>
      <height>18</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(230, 230, 230, 0);</string>
    </property>
    <property name="text">
     <string>44.1k</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="radio32">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>71</width>
      <height>18</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(230, 230, 230, 0);</string>
    </property>
    <property name="text">
     <string>32k</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radio22">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>71</width>
      <height>18</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(230, 230, 230, 0);</string>
    </property>
    <property name="text">
     <string>22.05k</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radio48">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>71</width>
      <height>18</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(230, 230, 230, 0);</string>
    </property>
    <property name="text">
     <string>48k</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="btnCreate">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>421</y>
     <width>251</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>&amp;Create</string>
   </property>
   <property name="shortcut">
    <string>Meta+C</string>
   </property>
  </widget>
  <widget class="QGraphicsView" name="graphicsView_2">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>425</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(230, 230, 230, 150);</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>526</x>
     <y>10</y>
     <width>81</width>
     <height>101</height>
    </rect>
   </property>
   <property name="title">
    <string>Channels</string>
   </property>
   <widget class="QRadioButton" name="radioMono">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>61</width>
      <height>18</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(230, 230, 230, 0);</string>
    </property>
    <property name="text">
     <string>Mono</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioStereo">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>61</width>
      <height>18</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(230, 230, 230, 0);</string>
    </property>
    <property name="text">
     <string>Stereo</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_4">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>140</y>
     <width>261</width>
     <height>176</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="title">
    <string>OT Attributes</string>
   </property>
   <widget class="QSlider" name="sliderGain">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>114</y>
      <width>151</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(0, 0, 0, 0);</string>
    </property>
    <property name="minimum">
     <number>-48</number>
    </property>
    <property name="maximum">
     <number>48</number>
    </property>
    <property name="singleStep">
     <number>1</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <property name="tickPosition">
     <enum>QSlider::NoTicks</enum>
    </property>
    <property name="tickInterval">
     <number>1</number>
    </property>
   </widget>
   <widget class="QLabel" name="lblGain">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>114</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(0, 0, 0, 0);</string>
    </property>
    <property name="text">
     <string>Gain</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="lblBPM">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>144</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(0, 0, 0, 0);</string>
    </property>
    <property name="text">
     <string>BPM</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QSlider" name="sliderBPM">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>144</y>
      <width>151</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(0, 0, 0, 0);</string>
    </property>
    <property name="minimum">
     <number>120</number>
    </property>
    <property name="maximum">
     <number>1200</number>
    </property>
    <property name="singleStep">
     <number>1</number>
    </property>
    <property name="pageStep">
     <number>40</number>
    </property>
    <property name="value">
     <number>500</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <property name="tickPosition">
     <enum>QSlider::NoTicks</enum>
    </property>
    <property name="tickInterval">
     <number>1</number>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="txtGainValue">
    <property name="geometry">
     <rect>
      <x>43</x>
      <y>114</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="txtBPMValue">
    <property name="geometry">
     <rect>
      <x>43</x>
      <y>144</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
   </widget>
   <widget class="QComboBox" name="dropQuant">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>84</y>
      <width>241</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="dropLoop">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>54</y>
      <width>241</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="dropStretch">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>24</y>
      <width>241</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="btnStop">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>421</y>
     <width>81</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>&amp;Stop</string>
   </property>
   <property name="shortcut">
    <string>Space</string>
   </property>
  </widget>
  <widget class="QProgressBar" name="progressBar">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>400</y>
     <width>231</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(230, 230, 230, 150);</string>
   </property>
   <property name="value">
    <number>0</number>
   </property>
   <property name="textVisible">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QComboBox" name="dropNormalize">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>350</y>
     <width>261</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>10</y>
     <width>81</width>
     <height>101</height>
    </rect>
   </property>
   <property name="title">
    <string>Bitrate</string>
   </property>
   <widget class="QRadioButton" name="radio16">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>51</width>
      <height>18</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(230, 230, 230, 0);</string>
    </property>
    <property name="text">
     <string>16bit</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="radio24">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>51</width>
      <height>18</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(230, 230, 230, 0);</string>
    </property>
    <property name="text">
     <string>24bit</string>
    </property>
   </widget>
  </widget>
  <widget class="QListWidget" name="listSlices">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>421</width>
     <height>401</height>
    </rect>
   </property>
   <property name="acceptDrops">
    <bool>true</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(230, 230, 230, 150);
selection-background-color: rgb(170, 170, 127);</string>
   </property>
   <property name="editTriggers">
    <set>QAbstractItemView::EditKeyPressed</set>
   </property>
   <property name="dragDropMode">
    <enum>QAbstractItemView::InternalMove</enum>
   </property>
   <property name="defaultDropAction">
    <enum>Qt::IgnoreAction</enum>
   </property>
   <property name="textElideMode">
    <enum>Qt::ElideRight</enum>
   </property>
  </widget>
  <widget class="QCheckBox" name="chkCreateOTFile">
   <property name="geometry">
    <rect>
     <x>443</x>
     <y>120</y>
     <width>250</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Create .ot file</string>
   </property>
  </widget>
  <widget class="QComboBox" name="dropFadeOut">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>380</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="dropFadeIn">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>380</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <zorder>btnAddSilence</zorder>
  <zorder>btnRemove</zorder>
  <zorder>btnAddWav</zorder>
  <zorder>btnPlay</zorder>
  <zorder>groupBox_3</zorder>
  <zorder>btnCreate</zorder>
  <zorder>graphicsView_2</zorder>
  <zorder>groupBox_2</zorder>
  <zorder>groupBox_4</zorder>
  <zorder>btnStop</zorder>
  <zorder>progressBar</zorder>
  <zorder>dropNormalize</zorder>
  <zorder>groupBox</zorder>
  <zorder>listSlices</zorder>
  <zorder>lblSliceCOunt</zorder>
  <zorder>chkCreateOTFile</zorder>
  <zorder>dropFadeOut</zorder>
  <zorder>dropFadeIn</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
